Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift

```diff
@@ -97,11 +97,13 @@ class MSMainViewController: UITableViewController, AppCenterProtocol {
       self.dbFileDescriptor = dbFile.withUnsafeFileSystemRepresentation { fileSystemPath -> CInt in
         return open(fileSystemPath!, O_EVTONLY)
       }
+#if TARGET_OS_IOS
       self.dbFileSource = DispatchSource.makeFileSystemObjectSource(fileDescriptor: self.dbFileDescriptor, eventMask: [.write], queue: DispatchQueue.main)
```

Comment:
I am not sure this has to be removed from Catalyst build, but I do not found way to fix it (runtime error)

Reply:
Added TODO for now